packages/node/bench/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bench",
   "version": "1.0.0",
   "main": "index.js",
-  "license": "MIT",
+  "license": "Apache-2.0",
```

Comment:
Do we need to update this one? Should this be `"private": true` instead?

Reply:
6f27e651c15315dc129c95c979757cc4e6cd6c40